clappr/clappr-ios
# Create PlayButton Plugin

⚠️ DO NOT MERGE THIS PR UNTIL [THIS PR](https://github.com/clappr/clappr-ios/pull/227) IS MERGED ⚠️

### Goal

Creating a PlayButton plugin to append in MediaControl plugins

### How to test

Run example and tests: 

Execute the video and make sure that you can play and pause the video.
